src/components/LogsContainer.vue

```diff
@@ -62,7 +62,7 @@
   flex
   justify-center
   w-full
-  bg-slate-700
+  bg-slate-800
```

Comment:
Should the logs container and the JsonInput components have a "lite" version so that way they can just inherit the default light and dark mode colors? Rather than always being "dark"

Reply:
I thought about that but since both of those are pretty heavy on the custom styling I figure if there's enough demand we can tackle those later. Personally I kinda like the visual contrast in the light theme

Reply:
I do to but only because I want everything dark lol. It would just be nice if these didn't have special styling.